58.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @param {string} s
+ * @return {number}
+ */
+var lengthOfLastWord = function (s) {
+  let result = s.trim().split(" ");
+  result = result[result.length - 1].length;
```

Comment:
It is always good to give variables/constants self-descriptive names so that readers can understand what data is in the variables/constants just by looking at the words. You can do this to make their names more self-descriptive.

```suggestion
  const words = s.trim().split(" ");
  const lastWordLength = words[words.length - 1].length;
```

Reply:
Check out [10 JavaScript Naming Conventions Every Developer Should Know](https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx)